runtime/src/bridge.rs

```diff
@@ -86,6 +91,36 @@ decl_module! {
             Ok(())
         }
 
+        /// Transfer an nft to a whitelisted destination chain. Source nft is locked in bridge account
+        /// rather than being burned.
+        #[weight = 195_000_000]
+        pub fn transfer_asset(origin,
```

Comment:
Is there any reason why not naming this function `transfer_nft` ?

Reply:
Asset over nft is the naming convention throughout the project. Inspired from Dan Forbes' naming convention to try to be more general. That being said I have no problem in changing this, let me know.